ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-release-4.9.yaml

```diff
@@ -1,12 +1,12 @@
 base_images:
   cli:
-    name: "4.9"
+    name: "4.10"
```

Comment:
This doesn't look like something we want to touch.

Reply:
Looks also to be the reason why it wasn't auto approved.

Reply:
edited the wrong file :/